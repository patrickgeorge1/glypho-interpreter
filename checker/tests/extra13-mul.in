-1293856120985761287512368
-238476512980061207836106741087324698372
