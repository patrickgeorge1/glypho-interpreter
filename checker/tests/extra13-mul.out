308554296030592546210023739506953582515953549411499905819464896
